packages/synapse-react-client/src/lib/containers/dataaccess/AccessRequirementDashboard.tsx

```diff
@@ -121,9 +127,6 @@ export function AccessRequirementDashboard(
           selectableTypes: [EntityType.PROJECT],
         }}
         show={showEntityFinder}
-        onClose={() => {
-          setShowEntityFinder(false)
-        }}
```

Comment:
Remove `onClose` prop from EntityFinderModal and instead just use `onCancel` -- in all cases, `onClose` prop was passed the same function as `onCancel`. Ensures the same behavior however the dialog is closed, e.g. ConfirmationDialog in EntityFinderModal appears when clicking "Cancel" or when clicking the "X" close button.